server/lib/filter.js

```diff
@@ -195,12 +195,23 @@ class FilterParser {
   applyQuery(sql) {
     // optionally call utility method to parse all remaining options as simple
     // equality filters into `_statements`
-    let limitCondition = this._parseLimit();
+    const limitCondition = this._parseLimit();
     this._parseDefaultFilters();
-    let conditionStatements = this._parseStatements();
-    let order = this._order;
+    const conditionStatements = this._parseStatements();
+    const order = this._order;
+    let statement;
+
+    // allows a developer to specify the position of the WHERE injection
+    // by default, it will simply concat
+    const hasWhereInjection = sql.includes('{{ WHERE }}');
```

Comment:
In my mind, I would be in favor of leaving `LIMIT` handling in the `FilterParser` library... you can think of it as a type of filter that simple filters out result beyond X rows.

`filters.setGroup()` is fine as an API.  I wonder if we'll hit a corner case where the grouping isn't set in the correct location properly...  We can always try it an see.  I had started this PR that way, and then switched because I worried it wouldn't turn out correctly.